test/testdrive/avro-unions.td

```diff
@@ -38,11 +40,10 @@ c1         true      bigint
 c2         true      text
 d1         true      bigint
 d2         true      text
-mz_obj_no  false     bigint
 
 > SELECT * FROM unions
-a   b       c1      c2     d1      d2      mz_obj_no
-----------------------------------------------------
-1   <null>  <null>  <null>  <null>  d      1
-2   2       <null>  foo     4       <null> 2
-2   2       3       <null>  <null>  d      3
```

Comment:
we can also include offset, but for this test it doesnt really matter

Reply:
Yeah this test didn't seem to care; was just including it because typing `SELECT *` was easiest.